client/client.go

```diff
@@ -58,6 +58,8 @@ type Client interface {
 	SendPasswordResetEmail(email string) (bool, *model.Response)
 	UpdateUser(user *model.User) (*model.User, *model.Response)
 	UpdateUserMfa(userId, code string, activate bool) (bool, *model.Response)
+	CreateUserAccessToken(userId, description string) (*model.UserAccessToken, *model.Response)
```

Comment:
should there be an api to revoke a token? by user or tokenid or something? thoughts?

Reply:
Agree, we could use `RevokeUserAccessToken` for this